easybuild/tools/filetools.py

```diff
@@ -583,11 +591,18 @@ def guess_patch_level(patched_files, parent_dir):
             if os.path.isfile(os.path.join(parent_dir, *tf2[level:])):
                 path_found = True
                 break
+	    else:
+		# If the file is missing, we create an empty file,
+		# but only if the old line is /dev/null
+		if patched_file_old in ['/dev/null']:
+		    open(os.path.join(parent_dir, *tf2[level:]), "a+")
```

Comment:
@wpoely86: ping? are you up for taking over this PR from @blappm to get this merged (in one way or another)?